src/commcare_cloud/fab/fabfile.py

```diff
@@ -221,6 +221,7 @@ def env_common():
     all = servers['all']
     proxy = servers['proxy']
     webworkers = servers['webworkers']
+    manage = servers.get('manage', webworkers[0])
```

Comment:
I might go with a consistent "django manage" throughout. So calling the role above `ROLES_DJANGO_MANAGE`, and calling this server group variable  `django_manage` and the optional ansible inventory group accessed on this line `django_manage` as well.

Reply:
Seems fine and updated, but at some point perhaps celery, webworkers, and pillowtop should be updated too